massdriver-application/outputs.tf

```diff
@@ -23,6 +23,11 @@ output "params" {
 }
 
 output "id" {
-  description = "Cloud ID for application IAM (AWS Role, GCP Service Account, Azure Service Account, etc)"
+  description = "Cloud ID for application IAM (AWS Role, GCP Service Account, Azure Managed Identity, etc)"
   value       = local.application_identity_id
 }
+
+output "identity" {
```

Comment:
This should probably be marked as `sensitive = true`

Reply:
Okay, I'll add that